src/cluster/multicast.cpp

```diff
@@ -23,6 +23,10 @@
 #include "cocaine/context.hpp"
 #include "cocaine/logging.hpp"
 
+#include "cocaine/traits/vector.hpp"
```

Comment:
Because context_tag has method with following argument_type:
```
 typedef boost::mpl::list<
            std::string,
            std::tuple<std::vector<asio::ip::tcp::endpoint>, unsigned int, graph_root_t>
        > argument_type;
```
So we need to be able unpack it.